### DISCUSSION TITLE
Direction of the Admin UI

### DISCUSSION THREAD

**Discussion by ChuckJonas** [keystonejs/keystone#6134] [Event Q&A Q3 2021]
### What is the long term vision for the Admin UI?

Right now, Admin UI seems like a `CRUD` application without the `R` component.  It works ok for managing "content" and prototyping applications, but is lacking when it comes to data display, working with larger datasets and customized UI/Navigation.

Is there any interest in moving towards more of a general purpose "foundation" for building applications around the keystone-core?

**Some features that I think would help get it there:**

- [Custom layouts/pages and more dynamic "view" modes](https://github.com/keystonejs/keystone/discussions/6133)
- Relation "Search" mode
- [Relation "Table" mode](https://github.com/keystonejs/keystone/discussions/6129)
- [Customizable Navigation](https://github.com/keystonejs/keystone/discussions/6131)
- Customizable Dashboard
- Custom Branding/theming
- Better Support for custom pages which can compose of existing Keystone Admin UI features

**Comment by raveling** [comment]
@ChuckJonas we discussed this in the recent Q&A event: 
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=760s
Thanks for bring this question to the community 🙌
